internal/knowledge/querier.go

```diff
@@ -72,7 +73,12 @@ func (q *Querier) queryInternal(ctx context.Context, cypherQuery string) (*Queri
 		return nil, translation.Query, err
 	}
 
-	logrus.Debugf("Found results in %dms", s.Execution/time.Millisecond)
+	execution_time := s.Execution / time.Millisecond
+
+	logrus.Debugf("Found results in %dms", execution_time)
+
+	metrics.GraphQueryTimeExecution.
+		WithLabelValues().Observe(float64(execution_time))
```

Comment:
you could use `s.Execution.Seconds() / 1000` (gives you a float, and prevents loosing definition from integer division)

Reply:
Good catch, thanks 
`s.Execution.Seconds() * 1000` for we want ms